cli/roger_push.py

```diff
@@ -467,90 +457,16 @@ def main(self, settings, appConfig, frameworkObject, hooksObj, args):
                         print(colored(json.dumps(resp.json(), indent=4), color))
                         container_task_id = self.utils.modify_task_id(task_id)
                         self.task_id.extend(container_task_id)
-                        if hasattr(resp, "status_code"):
-                            status_code = resp.status_code
```

Comment:
Just double checking that removing these two lines is intentional. Is this related to statsd?